offer-manager-domain/src/main/groovy/life/qbic/business/customers/update/UpdateCustomerInput.groovy

```diff
@@ -1,12 +1,23 @@
 package life.qbic.business.customers.update
+
+import life.qbic.datamodel.dtos.business.Customer
 /**
  * Input interface for the {@link UpdateCustomer} use case
  *
  * <detailed description>
  *
  * @since: 1.0.0
- * @author: Tobias Koch
+ * @author: Andreas Friedrich
```

Comment:
we agreed to leave out the author tag from the classes, so you don't need to update that. Unless you want to ;)